### ISSUE TITLE
[ADDITION] Snappy Driver Installer Origin

### ISSUE THREAD

**Issue by Kommprog** [Lissy93/awesome-privacy#86]
### Category

Utilities

### Software to be added

Snappy Driver Installer Origin is a portable Windows tool to install and update device drivers. It can be used offline to install drivers where Internet isn't available. No more searching for drivers after a clean install, just let Snappy Driver Installer Origin do it's thing and your job will be done in no time. The perfect technician's tool.

### Supporting Material

https://www.snappy-driver-installer.org/
https://www.glenn.delahoy.com/snappy-driver-installer-origin/

### Association Disclosure

Not associated in any way, just grateful user of it.

### Would you like to submit a PR?

Maybe?

### Please tick the boxes

- [X] You have filled out this form accurately, and to the best of your knowledge
- [X] You have indicated weather or not you are associated with the suggested addition
- [X] A similar submission has not already [been opened](https://github.com/Lissy93/awesome-privacy/issues?q=is%3Aissue) for this software/ service
- [X] You agree to the [code of conduct](https://github.com/Lissy93/awesome-privacy/blob/main/.github/CODE_OF_CONDUCT.md)

**Comment by Lissy93** [comment]
Do you have a link to the source? 

Since anything driver related needs admin privileges it'd be important for me to check out the code

**Comment by Kommprog** [comment]
> 
Yep, seems like its source code is hosted on SF, but there was [thread ](https://www.glenn.delahoy.com/forums/topic/move-the-sdio-project-to-github/)

**Comment by Kommprog** [comment]
> Do you have a link to the source?

> Edit, I did find [this](https://sourceforge.net/projects/snappy-driver-installer-origin/), but it looks like it only includes the built output, I did search GitHub and GitLab, but couldn't find it.

Yep, seems like its source code is really hosted only on SF, but there was [thread](https://www.glenn.delahoy.com/forums/topic/move-the-sdio-project-to-github/) questioning for moving it to github or gitlab.

**Comment by haseakash** [comment]
How to hide taskbar icon of SDI exe?
